### PR TITLE
style(ui): update link button styling

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

update link button styling

image

## Testing

Test locally

image

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/ui/src/components/TextLink/index.module.scss [hunk 1]
```diff
@@ -9,6 +9,12 @@
     color: var(--color-brand-default);
     text-decoration: none;
     font: var(--font-label-2);
+    border-radius: 4px;
+    padding: 4px 2px;
```
Comment (charIeszhao) [inline packages/ui/src/components/TextLink/index.module.scss:13]: nit: 
```suggestion
    padding: _.unit(1) _.unit(0.5);
```

File: packages/ui/src/components/TextLink/index.module.scss [hunk 2]
```diff
@@ -21,10 +27,15 @@
 :global(body.desktop) {
   .link {
     &.primary:hover {
-      text-decoration: underline;
+      background: var(--color-overlay-brand-hover);
+    }
+
+    &.primary:focus-visible {
+      border: _.border(var(--color-overlay-brand-focused));
```
Comment (charIeszhao) [inline packages/ui/src/components/TextLink/index.module.scss:34]: Should use outline instead of border

Comment (simeng-li) [reply packages/ui/src/components/TextLink/index.module.scss:34]: yes

Comment (simeng-li) [reply packages/ui/src/components/TextLink/index.module.scss]: updated

**Pull request merged**